HISTORY.md

```diff
@@ -1,5 +1,8 @@
 # History
 ---
+## 1.5.0 / 2017-02-10
+- add `onDoubleClick` API
```

Comment:
This feels a tiny bit odd, but I do see how this adds the double click functionality